releasenotes/notes/deprecate_assemble-67486b4d0a8d4f96.yaml

```
@@ -0,0 +1,10 @@
+---
+deprecations_providers:
+  - |
+    The structures related to Qobj are now deprecated. Interacting with remote backends should be done via QPY or
+    OpenQASM instead.
```

Not to be annoying but I would appreciate a full list of the classes being deprecated for future reference/being able to search for a specific deprecation in the reno.

```suggestion
    The `Qobj` structure and related classes are now deprecated, they were introduced as part of the `BackendV1` workflow and are no longer necessary for interacting with  :class:`.BackendV2` backends. Remote backend interaction should be done via `QPY` or `OpenQASM` instead.
```